### PR TITLE
Release 9.0.5

### PR DESCRIPTION
**Pull request by alyezz**
* Rollback unified initialization
* Update native iOS and android SDKs
* Update Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,28 +7,67 @@ Instabug is a bug reporting and in-app feedback tool that provides your testers
...
 ## Usage
-1. You can initialize the SDK by using this method in your JS class
+
+To initialize Instabug in your app, you need to do the following:
+
+### Android
+
+1. Change the name of the application class in your AndroidManifest.xml file to `android:name="com.instabug.cordova.plugin.MyApplication"`.
+
+Since `Cordova Android 7.0.0` AndroidManifest.xml position has changed and it is placed under `app/src/main/AndroidManifest.xml` ([read more](http://cordova.apache.org/announcements/2017/12/04/cordova-android-7.0.0.html)). You have to point it in your `.gradle` file in `platforms\android\com.instabug.cordova.plugin` directory in this way:
```
Comment (AliAbdelfattah) [inline README.md]:
```suggestion
Starting from _Cordova Android v7.0.0_ `AndroidManifest.xml` location has been changed and its new path now is `app/src/main/AndroidManifest.xml` ([read more](http://cordova.apache.org/announcements/2017/12/04/cordova-android-7.0.0.html)).

If you're using _Cordova Android v7+, please make sure you update its location inside Cordova's `build.gradle` file at `platforms/android/CordovaLib/build.gradle`:
```

File: README.md [hunk 2]
```diff
@@ -8,28 +7,67 @@ Instabug is a bug reporting and in-app feedback tool that provides your testers
...
+
+```
+sourceSets {
```
Comment (AliAbdelfattah) [inline README.md]:
```suggestion
    sourceSets {
```

File: README.md [hunk 3]
```diff
@@ -8,28 +7,67 @@ Instabug is a bug reporting and in-app feedback tool that provides your testers
...
+        main {
+            manifest.srcFile 'src/main/AndroidManifest.xml'
+        }
+    }
+```
+
+2. You need to add your app token in the **MyApplication** class, by replacing `YOUR_ANDROID_TOKEN`. (You can find this class under this path `YourProjectName/platforms/android/app/src/main/java/com.instabug.cordova.plugin/MyApplication.java`)
```
Comment (AliAbdelfattah) [inline README.md]: It seems the path has been changed to `plugins/instabug-cordova/src/android/MyApplication.java` and therefore no integration code was added at my end.

File: README.md [hunk 4]
```diff
@@ -8,28 +7,67 @@ Instabug is a bug reporting and in-app feedback tool that provides your testers
...
+4.  Make sure the following snippet is added to your project level `build.gradle`, if not you can manually add it as follows:.
+
+```dart
+allprojects {
+	repositories {
+	    maven {
+	        url "https://sdks.instabug.com/nexus/repository/instabug-cp"
+	    }
+	}
+}
```
Comment (AliAbdelfattah) [inline README.md]:
```suggestion
    allprojects {
	repositories {
	    maven {
	        url "https://sdks.instabug.com/nexus/repository/instabug-cp"
	    }
	}
    }
```

File: README.md [hunk 5]
```diff
@@ -38,20 +76,12 @@ cordova.plugins.instabug.activate(
...
-	}
-}
-```
- ##### TypeScript
-If you're using this plugin with TypeScript, you need to add this line at the beginning of your __app.component.ts__ class.
+You can change the invocation event with any of the following: `'button'`, `'screenshot'`, `'swipe'`, or `'shake'`.
+
+##### Ionic
+
+If you're using this plugin with Ionic, you need to add this line at the beginning of your **app.component.ts** class.
```
Comment (AliAbdelfattah) [inline README.md]:
```suggestion
##### TypeScript	
If you're using this plugin with TypeScript, you need to add this line at the beginning of your __app.component.ts__ class.
```

File: README.md (hunk 3 above)
Comment (alyezz) [reply README.md]: You will find it in the path mentioned in the Readme too.

**Pull request merged**